.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "testing-overhead/spring-petclinic-rest"]
```

Comment:
wat

Reply:
Yeah, good call...we don't have to do it this way.  We can just clone it into the base docker image instead.